### ISSUE TITLE
the API doesn't work

### ISSUE THREAD

**Issue by androidAppMe** [codelucas/newspaper#961]
Hi all. I was using newspaper3k and it was working fine, but today it stopped working and returns empty text. Does anyone have any ideas?

**Comment by iGetAlways** [comment]
me too having this problem, any fixes?

**Comment by GalKaplun** [comment]
same here

**Comment by iGetAlways** [comment]
i figured out that the api stopped working on google rss article links
it was working until 2nd february

**Comment by banagale** [comment]
Would you please provide a sample implementation or link to a Google rss feed that is now broken?So the error can be reproduced. On Feb 14, 2023, at 7:56 AM, Rıdvan ***@***.***> wrote:﻿
i figured out that the api stopped working on google rss article links
it was working until 2nd february

**Comment by ridvanpro** [comment]
> Would you please provide a sample implementation or link to a Google rss feed that is now broken?So the error can be reproduced. On Feb 14, 2023, at 7:56 AM, Rıdvan ***@***.***> wrote:﻿ i figured out that the api stopped working on google rss article links it was working until 2nd february —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

this is taken from google news rss which will redirect to original article link:
[https://news.google.com/rss/articles/CBMiLGh0dHBzOi8vd3d3LmZvbnRhbmthLnJ1LzIwMjMvMDIvMTQvNzIwNTk3MDIv0gEA?oc=5&hl=en-US&gl=US&ceid=US:en](url)

i scrape (getContent) this url via newspaper and get a top image which is `og:image` of the google news link which will redirect to original article link: [https://lh3.googleusercontent.com/J6_coFbogxhRI9iM864NL_liGXvsQp2AupsKei7z0cNNfDvGUmWUy20nuUhkREQyrpY4bEeIBuc=s0-w300-rw](url)

**Comment by androidAppMe** [comment]
Now I'm having a problem with "trafilatura" API as well. Can't get the body of the news with trafilatura as well, which was working finde before!

**Comment by ridvanpro** [comment]
> Now I'm having a problem with "trafilatura" API as well. Can't get the body of the news with trafilatura as well, which was working finde before!

are you using google rss too?

**Comment by androidAppMe** [comment]
Yes, I'm extracting the rss by pygooglenews. but I can't parse it. Could anybody find a solution? I tried to getting the news directly form google news page but it keeps blocking my IP.

**Comment by ridvanpro** [comment]
i didnt have issue with parsing google news, it was about google's redirect to original page. i solved the problem adding this before using newspaper's getcontent function:

```
import requests
import time
r = requests.get("google news url taken from google rss")
time.sleep(1)
#r.url is redirected url
```
